consumers/writers/README.md

```diff
@@ -0,0 +1,16 @@
+# Writers
+
+Writers provide an implementation of various `message writers`.
```

Comment:
Is this now rather implementation of `Consumer` interface?

Reply:
I would add a README for consumers and keep this one